signing/mcl/multisig/common.go

```diff
@@ -0,0 +1,134 @@
+package multisig
```

Comment:
Can we group these function in one (if just one, that would mean we just rename this one) or two files with related funcs? So we won't add this `common.go` file. It's an invitation to pile other unrelated stuff here